src/libs/actions/Report.js

```diff
@@ -820,6 +820,39 @@ function fetchAllReports(
         });
 }
 
+/**
+ * Creates an optimistic report with a randomly generated reportID and as much information as we currently have
+ *
+ * @param {Array} participantList
+ * @returns {Object}
+ */
+function createOptimisticReport(participantList) {
+    return {
+        chatType: '',
+        hasOutstandingIOU: false,
+        isOwnPolicyExpenseChat: false,
+        isPinned: false,
+        lastActorEmail: '',
+        lastMessageHtml: '',
+        lastMessageText: null,
+        lastReadSequenceNumber: undefined,
+        lastMessageTimestamp: 0,
+        lastVisitedTimestamp: 0,
+        maxSequenceNumber: 0,
+        notificationPreference: '',
+        oldPolicyName: '',
+        ownerEmail: '__FAKE__',
+        Participants: participantList,
```

Comment:
Is the capitalization intentional?

Reply:
nope, nice catch! Stupid google doc auto-capitalization